vue.config.js

```diff
@@ -9,6 +9,7 @@
 */
 
 const path = require('path');
+const webpack = require('webpack');
```

Comment:
eslint is throwing an error here, because webpack is a dev dep. Generally this can be safely ignored, with an eslint ignore comment.